docs/_static/didman/v1.yaml

```diff
@@ -302,3 +333,13 @@ components:
         website:
           type: string
           description: URL of the public website of this Service Provider. Can point to a Nuts specific page with more information about the node and how to contact.
+    SearchResult:
```

Comment:
I think I would prefer a custom search response with just a few fields like
```
{
  name: "Zorgcentrum Roodvink",
  location: "Hengelo",
  did: "did:nuts:123",
  services: [
  // list of resolved and relevant services
  ]
}
```

Reply:
I'll make it like:

```json
{
  organization: {
    ...
   }
  didDocument: {
    ...
  }
```

Reply:
I thought we had an `Organization` domain object but I was obviously confused with `Nuts Demo EHR`. I still think we should just return the concept here, since adding yet another abstraction won't necessarily make it simples for calling clients (who need to understand and be able to process every abstraction).

What I can do however, is return the "organization" field of the concept, so that it contains roughly the fields you suggest.